### PR TITLE
Initial PR for repo

### PR DESCRIPTION
**Pull request by bengo1990**
Please pull this down and test from the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+
+On line 2 in offlineVualto.js , replace the const token value to a token generated by the VUDRM admin site. Ensure that {“liccache”:”yes”} is added to the Policy before generating the token. This will enable the Widevine license to be used with offline content.
+
+To test offline DRM encrypted playback, https is needed due to Chrome using EME. The easiest way to set this up is to follow the below:
+
+- Ensure Visual Studio Code is installed.
+
+- Install the “Live Server” extension: https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer
+
+To enable https, you must create a certificate to self authenticate for local host:
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
To enable https, you must create a certificate to self authenticate for localhost: 
```

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+
+To enable https, you must create a certificate to self authenticate for local host: 
+
+- Create a private key - ensure openssl is installed and input the following command: openssl genrsa -aes256 -out localhost.key 2048 
+
+- Provide a password. This will create the localhost.key.
+
+- Create the certificate by inputting the following command: openssl req -days 3650 -new -newkey rsa:2048 -key localhost.key -x509 -out localhost.pem
+
+- Input the password from the above, complete all prompts.This creates locahost.pem
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
- Input the password from the above, complete all prompts. This creates locahost.pem.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+
+To test offline DRM encrypted playback, https is needed due to Chrome using EME. The easiest way to set this up is to follow the below:
+
+- Ensure Visual Studio Code is installed.
+
+- Install the “Live Server” extension: https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer
+
+To enable https, you must create a certificate to self authenticate for local host: 
+
+- Create a private key - ensure openssl is installed and input the following command: openssl genrsa -aes256 -out localhost.key 2048
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
- Create a private key - ensure openssl is installed and input the following command: `openssl genrsa -aes256 -out localhost.key 2048`
```

File: README.md [hunk 4]
```diff
@@ -1 +1,49 @@
...
+
+- Install the “Live Server” extension: https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer
+
+To enable https, you must create a certificate to self authenticate for local host: 
+
+- Create a private key - ensure openssl is installed and input the following command: openssl genrsa -aes256 -out localhost.key 2048 
+
+- Provide a password. This will create the localhost.key.
+
+- Create the certificate by inputting the following command: openssl req -days 3650 -new -newkey rsa:2048 -key localhost.key -x509 -out localhost.pem
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
- Create the certificate by inputting the following command: `openssl req -days 3650 -new -newkey rsa:2048 -key localhost.key -x509 -out localhost.pem`
```

File: README.md [hunk 5]
```diff
@@ -1 +1,49 @@
...
+    "enable": true,
+    "cert": "<location of localhost.pem>",
+    "key": "<location of localhost.key>",
+    "passphrase": "<password when creating the localhost.key>"
+    }
+}
+
+- You may get a warning saying the certificate is not secure. Click advanced and go continue to the site.
+
+Assuming everything has been set up correctly, you should now see the Shaka Player working locally with https
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
Assuming everything has been set up correctly, you should now see the Shaka Player working locally with https.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,49 @@
...
+
+By default, Content Name is set to “vualto-drm-encrypted-video”
```
Comment (PerritonBranch) [inline README.md]:
```suggestion
By default, Content Name is set to “vualto-drm-encrypted-video”.
```

**Review by PerritonBranch — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (bengo1990) [reply README.md]: Actioned.

File: README.md (hunk 2 above)
Comment (bengo1990) [reply README.md]: Actioned.

File: README.md (hunk 3 above)
Comment (bengo1990) [reply README.md]: Actioned.

File: README.md (hunk 4 above)
Comment (bengo1990) [reply README.md]: Actioned.

File: README.md (hunk 5 above)
Comment (bengo1990) [reply README.md]: Actioned.

File: README.md (hunk 6 above)
Comment (bengo1990) [reply README.md]: Actioned.

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**